File: src/targets/node/unirest/fixtures/jsonObj-null-value.js
const unirest = require('unirest');

<<<<<<< HEAD
const req = unirest('POST', 'https://httpbin.org/anything');
=======
const req = unirest('POST', 'http://mockbin.com/har');
>>>>>>> upstream/master

req.headers({
  'content-type': 'application/json'
});

req.type('json');
req.send({
  foo: null
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: src/targets/java/unirest/fixtures/jsonObj-null-value.java
<<<<<<< HEAD
HttpResponse<String> response = Unirest.post("https://httpbin.org/anything")
=======
HttpResponse<String> response = Unirest.post("http://mockbin.com/har")
>>>>>>> upstream/master
  .header("content-type", "application/json")
  .body("{\"foo\":null}")
  .asString();